frontend/public/extend/devconsole/components/ImportFlowForm/ImportFlowForm.tsx

```diff
@@ -67,6 +67,22 @@ const initialState: State = {
   isBuilderImageDetected: false,
 };
 
+enum UrlErrorType {
+  RepoNotReachable = 'RepoNotReachable',
+  BranchNotFound = 'BranchNotFound',
+}
+
+const getUrlErrorMessage = (errorType: UrlErrorType) => {
+  switch (errorType) {
+    case UrlErrorType.RepoNotReachable:
```

Comment:
You're not actually using enum here if you use it this way. This basically is 
```suggestion
    case 'RepoNotReachable`:
```
Because you need to have a case for every type or error.

Reply:
@rohitkrai03 the existing console code has it like what I have done

Reply:
Done